### PR TITLE
Added pull_request_template and issue_template

### PR DESCRIPTION
**Pull request by HarshalRathore**
Solves #7 
I request @CIPHERTron / @yochannah  to review my PR and if no further changes are required then please merge these changes or add `hacktoberfest-accepted` label to this PR before 31st Oct as it will be counted towards my 4PR count.
Thank you.
Signed-off-by: HarshalRathore <harshalrathore2014@gmail.com>

### CODE REVIEW

**Review by CIPHERTron — CHANGES_REQUESTED** [review]
Heyy @HarshalRathore, first of all, thanks for your contributions.
Second, the PR template looks good but the issue template that you've added here is outdated.
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates

You can take a look here ⬆️ 
This is the updated version of the issue templates that people tend to use nowadays 🙂

**Comment by HarshalRathore** [comment]
Hii @CIPHERTron
First thanks for adding `hactoberfest-accepted` label even before I submitted the requested changes.
So I checked out the new Issue forms I wanted to ask that do you only want me to create a single issue form template like only for `Bug-report` which is what I originally submitted or do you also want me to create a separate template for when people want to submit a `new-idea` or propose enhancement.
BTW this PR is showing under review for next 12 days so I think hactoberfest will count it only after the review period is over. Do you know something about this.

**Comment by HarshalRathore** [comment]
Hii @CIPHERTron 
I have added an Issue form template for `Report-a-bug` as you asked. please review it.
To see how this form actually looks like in action see [here](https://github.com/HarshalRathore/Hello-World/issues/new?assignees=&labels=bug%2Ctriage&template=report-a-bug.yml&title=%5BBug%5D%3A+)
I created a testing repository to test this issue form there it's [Hello-World](https://github.com/HarshalRathore/Hello-World) you can see this issue form there, try to file a bug issue there.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (JPTIZ) [inline .github/PULL_REQUEST_TEMPLATE.md:28]: I think these aren't relevant for our case, as we rely on Python installation with no AOT compiler toolchain needed. I would rather, if wanted, add Python/tooling versions:

```suggestion
**Test Configuration**:

* Python version:
* Poetry version:
```

File: .github/ISSUE_TEMPLATE/report-a-bug.yml
Comment (JPTIZ) [inline .github/ISSUE_TEMPLATE/report-a-bug.yml:36]: What do you think about rewording like this?

```suggestion
        Example: "It was expected that <X> happened."
        Or "Data returned should be in format <Y>."
```

File: .github/ISSUE_TEMPLATE/report-a-bug.yml
Comment (JPTIZ) [inline .github/ISSUE_TEMPLATE/report-a-bug.yml:53]: Oh wow :exploding_head: This issue submission by form is mind-blowing haha

**Comment by JPTIZ** [comment]
I think that's it for me, if @CIPHERTron doesn't have something else to add, after fixing them I consider it approved.

**Review by JPTIZ — APPROVED** [review]
I'll wait for a second approval :)

File: .github/ISSUE_TEMPLATE/report-a-bug.yml
Comment (CIPHERTron) [inline .github/ISSUE_TEMPLATE/report-a-bug.yml:66]: a small typo: **read**

File: .github/ISSUE_TEMPLATE/report-a-bug.yml
Comment (CIPHERTron) [inline .github/ISSUE_TEMPLATE/report-a-bug.yml:68]: how about removing this label as of now?

**Review by CIPHERTron — COMMENTED** [review]
@HarshalRathore thanks for the PR. Overall LGTM 🚀 
Just a couple of suggestions and then I can approve this PR

**Review by CIPHERTron — APPROVED** [review]
LGTM 🎉

**Comment by JPTIZ** [comment]
2 approvals, way to go :) Thank you for your contribution and @CIPHERTron for the proper and ready review!

**Comment by HarshalRathore** [comment]
Thank you @JPTIZ and @CIPHERTron. I was able to learn something new from this.

**Comment by CIPHERTron** [comment]
> Thank you @JPTIZ and @CIPHERTron. I was able to learn something new from this.

Thank you for your contributions ❤️ 

If you're interested in full-stack development do check out the repositories in [code-is-science](https://github.com/codeisscience) organization. Do let me know if I can help you anyway and don't forget to join our slack 
 where all the conversations take place 😃  

You can find the slack invite [here](https://codeisscience-community.netlify.app/)

**Pull request merged**
